packages/turf-circle/index.js

```diff
@@ -37,5 +31,5 @@ module.exports = function (center, radius, steps, units) {
 
     coordinates.push(coordinates[0]);
 
-    return polygon([coordinates], properties);
+    return polygon([coordinates]);
```

Comment:
@rowanwins might need to revert this commit (or change it back to it's previous state).